terraform/helm/fullnode/values.yaml

```diff
@@ -39,10 +39,10 @@ image:
 
 resources:
   limits:
-    cpu: 7
+    cpu: 6.5
```

Comment:
nit: probably keep the CPU limit at 7, and only change the request

Reply:
resolved